src/cljs/rems/application.cljs

```diff
@@ -719,38 +719,126 @@
    {:type "button" :data-toggle "modal" :data-target "#review-request-modal"}
    (text :t.actions/review-request)])
 
+;;; Actions tabs
+
+(defn- action-tab [action-name text]
+  (let [action-name (str "actions-" action-name)]
+    [:a.nav-item.nav-link
+     {:id (str action-name "-tab") :data-toggle "tab" :href (str "#" action-name) :role "tab" :aria-controls action-name :aria-selected false}
+     text]))
+
+(defn- approve-tab []
+  [action-tab "approve" (text :t.actions/approve)])
+
+(defn- reject-tab []
+  [action-tab "reject" (text :t.actions/reject)])
+
+(defn- return-tab []
+  [action-tab "return" (text :t.actions/return)])
+
+(defn- review-tab []
+  [action-tab "review" (text :t.actions/review)])
+
+(defn- third-party-review-tab []
+  [action-tab "3rd-party-review" (text :t.actions/review)])
+
+(defn- close-tab []
+  [action-tab "close" (text :t.actions/close)])
+
+(defn- withdraw-tab []
+  [action-tab "withdraw" (text :t.actions/withdraw)])
+
+(defn- review-request-tab []
+  [action-tab "review-request" (text :t.actions/review-request)])
+
+(defn- action-comment [label-title]
+  [:div.form-group
+   [:label {:for "judge-comment"} label-title]
+   [textarea {:id "judge-comment"
+              :name "judge-comment" :placeholder "Comment"
```

Comment:
Translation missing.

Reply:
Good catch, it's properly been that way for a long time.